cluster/gce/config-default.sh

```diff
@@ -106,6 +106,13 @@ ENABLE_L7_LOADBALANCING="${KUBE_ENABLE_L7_LOADBALANCING:-glbc}"
 #   standalone     - Heapster only. Metrics available via Heapster REST API.
 ENABLE_CLUSTER_MONITORING="${KUBE_ENABLE_CLUSTER_MONITORING:-influxdb}"
 
+# Historically fluentd was a manifest pod and then was migrated to DaemonSet.
+# To avoid situation during cluster upgrade when there are two instances
+# of fluentd running on a node, kubelet need to mark node on which
+# fluentd in not running as a manifest pod with appriopriate label.
+# TODO(piosz): remove this in 1.8
+NODE_LABELS="${KUBE_NODE_LABELS:-alpha.kubernetes.io/fluentd-ds-ready=true}"
```

Comment:
It's not a problem with backward compatibility, because `$KUBE_NODE_LABELS` hasn't existed before. I can add a comment here (or below near logging setup) that if you want to override the flag and use fluentd for logging at the same time you need to make sure that there is the label `alpha.kubernetes.io/fluentd-ds-ready=true` specified there. WDYT?